services/secret-service/doc/openapi.yaml

```diff
@@ -299,22 +339,87 @@ components:
         - clientSecret
         - refreshTokenUrl
       properties:
-        name:
-          type: string
         clientId:
           type: string
         clientSecret:
           type: string
-        refreshTokenUrl:
```

Comment:
Why was it removed? We have a requirement to refresh tokens automatically.

Reply:
It is moved to `endpoints.token`, see above.

Reply:
The naming is not self-explanatory

Reply:
Most providers have the same URL for code and refresh token requests. Using token only as identifier (see endpoints) is more generic